community/duo_unix/APKBUILD

```diff
@@ -1,39 +1,27 @@
-# Contributor: Alan Lacerda <alacerda@alpinelinux.org>
-# Maintainer: Paul Morgan <jumanjiman@gmail.com>
+# Contributor: Kris Reese <ktreese@gmail.com>
+# Maintainer: Kris Reese <ktreese@gmail.com>
```

Comment:
Not sure you need to remove previous contribution

Reply:
Oversight on my part.  Corrected in 738f14a203